spec/controllers/application_controller_spec.rb

```diff
@@ -37,5 +31,11 @@ def index
 
       expect(Sentry).to have_received(:set_user).with(id: user.id)
     end
+
+    it "sets caching headers" do
+      get :index
+
+      expect(response.headers).to include "Cache-Control" => "no-store"
+    end
```

Comment:
This spec isnt specific to whether they are signed in or not, so I think its fine outside of the contexts?

Reply:
ah yes - I'm going to duplicate this test for both signed in, and not signed in - to ensure we have the correct ordering of before actions.